packages/driver/src/cy/commands/navigation.coffee

```diff
@@ -98,7 +98,7 @@ specifyFileByRelativePath = (url, log) ->
     }
   })
 
-aboutBlank = (win) ->
+aboutBlank = (cy, win) ->
```

Comment:
Looks like this call to this function needs updating https://github.com/cypress-io/cypress/blob/isolated-runner-fixes/packages/driver/src/cy/commands/navigation.coffee#L828:L828